docs/administrators/operational_procedures_with_lsm.rst

```diff
@@ -1,200 +1,206 @@
-    4. If the API update is irreversible or the end-result after revert is different from the expected result, restore the database tables `lsm_serviceentity` and `lsm_serviceinstance`.
+Upgrade abort/revert
++++++++++++++++++++++++
 
-    Deployment of a new service model to the orchestrator
-    ########################################################
+1. Delete the bad (latest) version produced during the update in the web-console
+2. Push a revert commit onto the release branch (`git revert HEAD; git push`)
+3. Go through the Upgrade procedure again to make this revert effective
+4. If the API update is irreversible or the end-result after revert is different from the expected result, restore the database tables `lsm_serviceentity` and `lsm_serviceinstance`.
 
-    This process describes how to safely deploy a new model to the orchestrator.
+Deployment of a new service model to the orchestrator
+########################################################
 
-    Context
-    ++++++++
-    * The orchestrator has an environment set up for the project, but it has not been deployed yet.
-    * The project is released (as described above)
+This process describes how to safely deploy a new model to the orchestrator.
 
-    Procedure
-    ++++++++++
+Context
+++++++++
+* The orchestrator has an environment set up for the project, but it has not been deployed yet.
+* The project is released (as described above)
 
-    1. Cross check all settings in the environment settings tab with the development team.
-    2. Verify with the development team what the correct entry point is for exporting the API definition.
-       Exporting the api definition requires the model to compile.
-       Often, the model requires a correct API definition to compile.
-       To break this cycle, developers have to provide an alternative entry point into the code (other than `main.cf`) that loads only the definitions.
-       We will assume this file is called `loader.cf`
+Procedure
+++++++++++
 
-    3. Verify that environment safety settings are on (should always be the case)
+1. Cross check all settings in the environment settings tab with the development team.
+2. Verify with the development team what the correct entry point is for exporting the API definition.
+   Exporting the api definition requires the model to compile.
+   Often, the model requires a correct API definition to compile.
+   To break this cycle, developers have to provide an alternative entry point into the code (other than `main.cf`) that loads only the definitions.
+   We will assume this file is called `loader.cf`
 
+<<<<<<< HEAD
        * `purge_on_delete = False`
        * `protected_environment = True`
+=======
+3. Verify that environment safety settings are on (should always be the case)
+>>>>>>> f164a1ab8 (Fix bug where clicking a sub-element from the ``Inmanta Lifecycle Service Manager`` menu would collapse the whole menu.)
```

Comment:
You forgot to resolve this merge conflict.

Reply:
Damn it, sorry about that